notebook/services/kernels/tests/test_kernels_api.py

```diff
@@ -63,29 +63,33 @@ def test_default_kernel(self):
         # POST request
         r = self.kern_api._req('POST', '')
         kern1 = r.json()
-        self.assertEqual(r.headers['location'], '/api/kernels/' + kern1['id'])
+        self.assertEqual(r.headers['location'], url_path_join(self.url_prefix, 'api/kernels', kern1['id']))
```

Comment:
Here you aren't escaping `kern1['id']`, is that just because you know the format of the kernel id doesn't need to be escaped?